third_party/jetifier/jetifier-standalone/java/com/github/bazelbuild/rules_jvm_external/JetifierWorkerWrapper.java

```diff
@@ -0,0 +1,46 @@
+package com.github.bazelbuild.rules_jvm_external;
```

Comment:
This file should live under `//private/tools/java` rather than in a new top-level directory.